client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/style.scss

```diff
@@ -1,6 +1,10 @@
 @import "@wordpress/base-styles/mixins";
 @import "@wordpress/base-styles/breakpoints";
 
+:root {
+	--alert-yellow: #f0b849;
+	--alert-yellow-light: #fef8ee;
+}
```

Comment:
Can we add an empty line above and below? That is how ideally it should be. I think there is some linting issues.

Reply:
I think I got styllnt fixed (though probably broke my PHP linting).  Anyway, the CSS has been moved to its own file supporting the component.